homeassistant/components/suez_water/quality_scale.yaml

```diff
@@ -49,24 +51,32 @@ rules:
   entity-translations: done
   entity-device-class: done
   devices: done
-  entity-category: todo
-  entity-disabled-by-default: todo
+  entity-category:
+    status: done
+    comment: only data entities
+  entity-disabled-by-default:
+    status: done
+    comment: only two data entities
   discovery:
     status: exempt
     comment: api only, nothing on local network to discover services
   stale-devices:
     status: exempt
-    comment: one device only
+    comment: one permanent device only
   diagnostics: todo
   exception-translations: todo
-  icon-translations: todo
-  reconfiguration-flow: todo
+  icon-translations:
+    status: exempt
+    comment: no custom icons
+  reconfiguration-flow:
```

Comment:
this has a couple of installation parameters, why would it be exempt?

Reply:
Only installation parameters are username and password which will be handled by reconfiguration flow.
And counter id which another PR is currently under review to be used as unique_id.

Reply:
And this item talks about reconfiguration, not options, so todo

Reply:
In that case I kind of fail to see the difference. Can you enlighten me?
The only thing configurable for now are: username / password => which are gonna be handled by reauth and counter_id which isn't updatable and will be auto-discovered (I hope) with the discussion about the devices

Reply:
The difference is that reconfiguration can be triggered by the user while reauth cannot. So it makes sense to have both, they can maybe share the same code though.

Reply:
Oooh finally understood, I really fought reconfiguration was for thing not in reauth